python/pyspark/ml/param/__init__.py

```diff
@@ -375,6 +375,18 @@ def copy(self, extra=None):
         that._defaultParamMap = {}
         return self._copyValues(that, extra)
 
+    def set(self, param, value):
+        """
```

Comment:
This does look extraneous.  I'm OK with adding it to match the Scala API, but it'd be better to put it in a separate PR and JIRA.